### ISSUE TITLE
Scroll to Bottom in Messages for Deafult

### ISSUE THREAD

**Issue by sunnymiu** [mfkp/tinderplusplus#36]
Currently the message section seem like a bit unpolished... features can include

- auto scroll to bottom when entering conversations 
- support for emojis
- highlight (and thus copying) on sent messages
- reduce message bubble size. Currently seem like 2x for for each line
- place text box in the bottom as always. Currently short conversation will cause the box placed on the middle
- ability to unmatch on conversation list

**Comment by sunnymiu** [comment]
.

**Comment by mrmayfield** [comment]
Agreed. Emojis and copy/paste are waiting in a current PR but I think that autoscroll to bottom would be huge.

**Comment by mrmayfield** [comment]
Scroll to bottom is implemented in a PR as well. Agreed on umatching though.
